src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -392,8 +394,6 @@ TransactionManager::do_submit_transaction(
 	ceph_assert(0 == "Hit error submitting to journal");
       })
     );
-  }).finally([&tref]() {
-      tref.get_handle().exit();
```

Comment:
Looks `exit()` is missing from `submit_transaction_direct()` after the change?

Reply:
Yeah, just fixed it.